docs/metasploit-framework.wiki/Metasploit-Guide-Session-Search.md

```diff
@@ -0,0 +1,60 @@
+## Session Search
```

Comment:
```suggestion
## Sessions Command

### Session Search
```

If we move towards this page being a more holistic 'Sessions Command' documentation, it means in the future we could add additional subsections like:

```

## Sessions Command

### Session Search

...

### Session interaction

...


### Upgrading sessions

...

```